src/Common/Commands.Common/MetricHelper.cs

```diff
@@ -207,9 +207,11 @@ public void FlushMetric()
         /// <returns></returns>
         public static string GenerateSha256HashString(string originInput)
```

Comment:
@markcowl is null check necessary here? or is it getting null checked before calling this function.
Would be good to skip if "originInput" is null